src/commands/schema/status.mjs

```diff
@@ -48,19 +63,35 @@ async function doStatus(argv) {
   }
 
   // Output the status response
-  logger.stdout(`Staged changes: ${chalk.bold(statusResponse.status)}`);
-  if (statusResponse.pending_summary !== "") {
-    logger.stdout(statusResponse.pending_summary);
-  }
-  if (statusResponse.diff) {
-    logger.stdout("Staged changes:\n");
-    logger.stdout(statusResponse.diff.split("\n").join("\n  "));
+  switch (statusResponse.status) {
+    case "none":
+      logger.stdout(`No staged changes${formatDatabaseName(argv.database)}.`);
+      break;
+    case "pending":
+    case "ready":
+      logger.stdout(
+        `Staged changes${formatDatabaseName(argv.database)} are ${chalk.bold(statusResponse.status)}:`,
+      );
+      if (statusResponse.status === "ready" && statusResponse.diff) {
+        logLineWithTab("(use `fauna schema commit` to commit staged changes)");
+        logBlockWithTab(statusResponse.diff);
+      } else if (statusResponse.pending_summary) {
+        logBlockWithTab(statusResponse.pending_summary, 1);
+      }
```

Comment:
I like how the diff is displayed for local changes and ready staged changes. Do you think there is a clear way to include the diff when reporting pending changes?